### ISSUE TITLE
Highlight units hotkey (F) - unit highlight does not turn off

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#12707]
### Reproduction Steps

1. Start any game, 2.6 (2.6+14780)
2. Press 'f' key (possibly needs to be pressed a couple times to trigger)
3. Notice: unit highlight does not go away

### Engine Version

 (2.6+14780)

### Expected Behavior

Prior the unit highlight was (generally) temporary. There were a few bugs that could be triggered that would keep the highlight on., but otherwise it was on a delay.

I would expect either for:
- highlight to be temporary
- stay on until 'f' is pressed again

**Comment by Cernelius** [comment]
I never use this feature and wonder how many do.

**Comment by DanVanAtta** [comment]
@Cernelius  we risk going off topic. TripleA really could use a feature-usage tracker.. Until we have data, it's not useful to discuss usage like that.

**Comment by TheDog-GH** [comment]
Observation
Press f once highlights own **moveable** units as expected.
Press f again does not turn off the highlight.
If I move the mouse very slowly it stays on.
If I move the mouse quickly it turns off.
Hope this helps?

**Comment by asvitkine** [comment]
Looks like there's no code logic for 'f' to unhighlight units. But some other stuff (e.g. if you're in a move phase) would reset highlighting. We could make it a toggle.

**Comment by DanVanAtta** [comment]
Big thank you for the fix @asvitkine 

@Cernelius sorry for shutting down the larger discussion - which is valid for you to bring up. I did not want us to "rabbit-hole" on that larger topic before we fixed the regression. 

I still think the 'F' key suffers from having no UI icon to make it easily discovered. It's hard to say  whether low usage is due to low utility or lack of discoverability (ie: people don't know it exists).  This does underscore the need to get some sort of usage metrics up and running. 'F' key is tough, as it can be really improved and cleaned up, perhaps do things like highlight factories during production & purchase. Though... even better would be to have a UI to toggle the map view, for example to highlight current faction, show normal colors, show allies, show production zones, etc...

Just wanted to go on the record a bit on where I think things should go -  but also underscore that without usage metrics it's tough to prune features which we think are unused. Pruning features is very healthy for a codebase, TripleA could use it where clearly reasonable.
